src/include/sof/drivers/ipc.h

```diff
@@ -126,7 +125,7 @@ static inline uint64_t ipc_task_deadline(void *data)
 
 static inline int32_t ipc_comp_pipe_id(const struct ipc_comp_dev *icd)
 {
-	extern tr_ctx ipc_tr;
+	extern struct tr_ctx ipc_tr;
```

Comment:
Btw, is there any plan to move this into SOF context ? fyi @mmaka1 @tlauda

Reply:
@lgirdwood I'm not sure what you are asking for.

Reply:
Given that these variables are in custom sections, I don't think they themselves can be in the SOF context (but maybe you _could_ have pointers?)